### PR TITLE
Day 4

### PR DESCRIPTION
**Pull request by MattGomb**
In this PR I finished the functions by:

- Adding the updated text-filed to localstorage.

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @MattGomb,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### - Your linters are not running on this PR
### - Ensure you completely gitignore `node_modules` so that it does not show up in the files changed as this makes reviews almost impossible to perform

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MattGomb** [comment]
@julie-ify hi, can you help me with the linters..? I did what i always do, but they dont run for some reason... is there a special configuration because of webpack? Thanks!!

**Comment by rloterh** [comment]
Hi Matt, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has `node_modules` directory included in the project structure. Kindly ensure to exclude this directory from uploading to the repository by including the directory in the `.gitignore` file.

You can try to clear git cache by running the commands in sequence in the terminal:
`rm -rf node_modules`
`npm cache clean --f`
`npm install`


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MattGomb** [comment]
@rloterh Hi, can you please explain what you mean? Because when i look at the branch, there are no node_module files, and i have the .gitignore file there, that contains the node_modules. Can you check again, please?
image

**Comment by rloterh** [comment]
Hi @MattGomb 

From the project structure below, it appears to be lingering:
image

But not actually there.

Kindly resubmit your project to be reviewed.

Cheers! 😃

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```
Comment (tresorsawasawa) [inline .eslintrc.json:25]: - [ **OPTIONAL** ] In `index.js` file:
Screenshot from 2022-10-08 01-25-50
Kindly make sure that the user is still able to delete a unique task to match the expected functionalities. Right now the user can not delete a task he wants if he doesn't use the `Clear All completed` button and for that, it requires that the task is completed.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @MattGomb 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
- [ **OPTIONAL** ] Kindly make sure you correct your pull request title has a meaningful name describing what changes are in the project for best practices. This should make your project to be invalidated but as the previous reviewer didn't mention it I suggest you update the title name.
- [ **OPTIONAL** ] Same thing for your branch name. Make sure for your next project you use `meaningful name` for your branches. A good branch should have a name of a `feature` or `action` (e.g. 'create_sidebar' or 'add_comments_section', 'add_interactive_list').
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json (hunk 1 above)
Comment (MattGomb) [reply .eslintrc.json:25]: Thank you for this comment! I fixed a linter in a way that broke this functionality, thank you for bringing it to my attention! It is now fixed, and delete button works! 😸

**Pull request merged**